### PR TITLE
Added typescript definitions to js module

### PR DESCRIPTION
**Pull request by daniil-sloboda**
* Added typescript definitions generation to output of the compilation script.     
* Updated package.json version

### CODE REVIEW

**Comment by daniil-sloboda** [comment]
@skearnes Hello, can you please take a look at the PR when you have some spare time?

File: js/ord-schema/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ord-schema",
-  "version": "0.3.94",
+  "version": "0.3.95",
```
Comment (skearnes) [inline js/ord-schema/package.json:3]: you should not update the version manually; bump2version handles this: https://github.com/open-reaction-database/ord-schema/blob/main/.bumpversion.cfg

File: compile_proto_wrappers.sh [hunk 2]
```diff
@@ -21,4 +21,6 @@ protoc \
   --python_out=. \
   --pyi_out=. \
   --js_out=import_style=commonjs,binary:js \
-  ../ord-schema/proto/*.proto
+  ../ord-schema/proto/*.proto \
+  --plugin="protoc-gen-ts=./js/ord-schema/node_modules/.bin/protoc-gen-ts" \
```
Comment (skearnes) [inline compile_proto_wrappers.sh:25]: If you can, please update this to remove `--plugin` and assume that protoc-gen-ts is in the path, like we do with JS: https://github.com/open-reaction-database/ord-schema/blob/main/.github/workflows/run_tests.yml#L88 (and update the GitHub workflow to add it)

Comment (daniil-sloboda) [reply compile_proto_wrappers.sh:25]: Thanks a lot for the explanation, I will update the PR tomorrow then

File: js/ord-schema/package.json (hunk 1 above)
Comment (daniil-sloboda) [reply js/ord-schema/package.json:3]: Okay, will revert the changes

**Comment by skearnes** [comment]
Thanks @daniil-sloboda this looks great!

**Review by skearnes — APPROVED** [review]

**Pull request merged**